default/scripts/asaDeploy.js

```diff
@@ -0,0 +1,69 @@
+const { balanceOf, executeTransaction } = require('@algo-builder/algob');
```

Comment:
how about renaming *asaDeploy.js* to *deploy-asa.js*?

Reply:
sure.